docs/src/stylesheets/theme/alerts.scss

```diff
@@ -0,0 +1,54 @@
+:root {
```

Comment:
let's exclude stylecheets from typo CI (see `.typo-ci.yml` file at the root of the repo)

Reply:
Nice catch. I'll exclude `*.jsx` too then if that's OK. In any case if I extract all this, it will be within the dependency

Reply:
...although, I notice that `*.md` files are specifically excluded too. I wonder what's it actually linting then? You probably want to *not* ignore `*.md` and `*.mdx` no?